code/modules/projectiles/guns/projectile/revolver/wayfarer.dm

```diff
@@ -0,0 +1,24 @@
+//Sprite by Rebel
+/obj/item/gun/projectile/revolver/wayfarer
+	name = "\"Wayfarer\" caseless revolver"
+	desc = "A exstreamly well hand made revolver of a mix of steel, iron and plasteel with a polished wooden stock. \
+	The barrel inside it refined to be ideal for 10x24 caseless and easy to clean. \
+	Well produced by Excelsior it dosnt have the normal markings other then the quote - \
```

Comment:
Here

Reply:
"While seemingly produced by Excelsior; hinted by the stamp placed upon its handguard, it lacks most of the normal markings - and is instead tagged with a simple quote -"